met-api/migrations/versions/c656f3f82334_add_languages_and_tenant_mapping.py

```diff
@@ -0,0 +1,788 @@
+"""
+Add the list of supported languages and create a table to map them to tenants when selected.
+
+Revision ID: c656f3f82334
+Revises: 22fb6b5b5aed
+Create Date: 2024-05-28 15:45:56.151488
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'c656f3f82334'
+down_revision = '22fb6b5b5aed'
+branch_labels = None
+depends_on = None
+
+language_data = [
+    {
+      "language": "Abkhazian",
+      "iso_code": "ab"
+    },
+    {
+      "language": "Afar",
+      "iso_code": "aa"
+    },
+    {
+      "language": "Afrikaans",
+      "iso_code": "af"
+    },
+    {
+      "language": "Akan",
+      "iso_code": "ak"
+    },
+    {
+      "language": "Albanian",
+      "iso_code": "sq"
+    },
+    {
+      "language": "Amharic",
+      "iso_code": "am"
+    },
+    {
+      "language": "Arabic",
+      "iso_code": "ar"
+    },
+    {
+      "language": "Aragonese",
+      "iso_code": "an"
+    },
+    {
+      "language": "Armenian",
+      "iso_code": "hy"
+    },
+    {
+      "language": "Assamese",
+      "iso_code": "as"
+    },
+    {
+      "language": "Avaric",
+      "iso_code": "av"
+    },
+    {
+      "language": "Avestan",
+      "iso_code": "ae"
+    },
+    {
+      "language": "Aymara",
+      "iso_code": "ay"
+    },
+    {
+      "language": "Azerbaijani",
+      "iso_code": "az"
+    },
+    {
+      "language": "Bambara",
+      "iso_code": "bm"
+    },
+    {
+      "language": "Bashkir",
+      "iso_code": "ba"
+    },
+    {
+      "language": "Basque",
+      "iso_code": "eu"
+    },
+    {
+      "language": "Belarusian",
+      "iso_code": "be"
+    },
+    {
+      "language": "Bengali (Bangla)",
+      "iso_code": "bn"
+    },
+    {
+      "language": "Bihari",
+      "iso_code": "bh"
+    },
+    {
+      "language": "Bislama",
+      "iso_code": "bi"
+    },
+    {
+      "language": "Bosnian",
+      "iso_code": "bs"
+    },
+    {
+      "language": "Breton",
+      "iso_code": "br"
+    },
+    {
+      "language": "Bulgarian",
+      "iso_code": "bg"
+    },
+    {
+      "language": "Burmese",
+      "iso_code": "my"
+    },
+    {
+      "language": "Catalan",
+      "iso_code": "ca"
+    },
+    {
+      "language": "Chamorro",
+      "iso_code": "ch"
+    },
+    {
+      "language": "Chechen",
+      "iso_code": "ce"
+    },
+    {
+      "language": "Chichewa, Chewa, Nyanja",
```

Comment:
I assume we are intentionally remaining neutral in the cases where languages have multiple names

Reply:
Nevermind; this is specified in the ticket